docs/about/join.md

```diff
@@ -41,24 +41,33 @@ To apply for an account on the Wynton HPC cluster, please fill out the following
  
  <div class="form-group">
   <label for="email_address">UCSF email address</label> <span style="color: red">*</span><br>
-  <input type="email" id="email" name="email" autocomplete="email" placeholder="Enter email" size="30" required><br>
+  <input type="email" id="email" name="email" autocomplete="email" placeholder="Enter email" size="30" required><br> 
+  <small>(If external collaborator, please provide preferred email id. Also, request your PI to send an email to <a href="mailto:wynton_admin@ucsf.edu">wynton_admin@ucsf.edu</a> approving the request and duration of access)</small><br>
```

Comment:
It might not be clear to the person who will submit this form when they would send that email request, e.g. they may wonder if they should send the request first and then submit the form.  Clarify that they should ask the PI after they've submitted the form.